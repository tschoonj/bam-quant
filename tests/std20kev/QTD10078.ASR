File: C:\xrfdaten\2014KW30\chris\std20kev\std10078.spe
Start: 21.07.2014 18:32:43
Stop: 21.07.2014 18:34:23
BESSY: 298.5274938 298.5354909
IONI: 3.033556E-11 0.000000000030537
ScanMotor1 exp_x: 1 1
ScanMotor2 exp_y: 1 1
Live: 84111
Run: 100000
Dead: 16
ICR: 49270
OCR: 0
Accumulated BESSY: 59703.3359274
Accumulated Ioni: 6.09251103E-09
N Accumulation: 200
Normfactor:  3.9028411e+008
E_cal:      0.020000000
E_cal_offset:      0.000000
Comments: Pt
exp_x 91.2603988647461 91.2604
exp_y 65.3237991333008 65.3238
exp_z 53.8899993896484 53.89
exp_w 135 135
exp_Tisch 870.6 870.600001928
dcm_theta 0 0
det_y 52.85 52.85
dmm_energy 20 20
dcm_energy 23 -184976.74536672
S3_horSize 0.03 3.00000000000001E-02
S3_verSize 0.03 3.00000000000001E-02
$PEAKS:   
      1
  78   2      9.434   1715009.      1223.     227.83
