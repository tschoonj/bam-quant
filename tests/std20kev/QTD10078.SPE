$DATA:
0 2000
19611
12523
5481
1734
332
48
9
1
4
5
2
0
2
0
0
4
154
1013
1245
1091
1026
1051
926
857
930
870
832
822
755
739
718
661
700
673
665
614
579
584
570
548
560
536
535
514
465
518
494
496
535
493
512
477
476
486
496
451
407
502
442
427
401
422
417
463
387
425
361
424
412
416
385
365
378
355
353
383
395
369
419
380
403
427
416
381
360
377
386
391
368
384
391
346
376
410
474
470
550
720
1022
1479
2145
2822
3230
3564
3372
3185
2757
2109
1428
1053
691
529
473
436
430
459
450
442
428
374
341
362
303
307
317
342
363
314
332
365
367
387
387
363
341
398
390
379
333
349
321
309
336
360
373
360
402
360
373
378
369
314
314
304
309
316
272
302
314
295
292
284
280
313
283
257
251
275
299
289
291
289
267
250
295
305
251
296
246
278
243
270
280
282
278
258
259
264
262
244
271
252
266
273
311
267
270
281
265
277
277
261
278
269
251
292
289
253
257
255
250
277
250
266
256
223
270
274
274
259
265
261
259
288
248
265
256
251
218
250
260
259
265
273
298
277
291
258
259
266
280
250
281
239
265
285
277
273
271
267
267
251
257
276
303
271
272
260
293
249
289
279
248
304
274
294
284
287
309
281
276
290
270
260
291
242
273
284
286
273
302
292
260
274
297
259
269
275
282
264
289
289
299
297
303
280
275
296
290
278
273
251
288
285
284
269
265
284
298
298
311
282
286
326
351
373
361
381
351
363
360
327
302
336
320
295
320
282
313
331
304
304
266
315
287
295
292
312
301
334
328
272
326
311
326
333
306
346
321
357
343
358
327
346
323
333
313
357
332
332
320
325
326
348
318
346
324
329
374
397
367
396
391
404
431
398
459
505
553
593
601
603
599
625
544
479
445
465
421
411
451
443
427
420
461
494
455
538
593
633
809
1050
1305
2020
2788
3902
5055
6326
7305
7674
7537
7001
6112
4951
3643
2606
1944
1387
1061
891
721
715
678
701
655
652
683
655
709
641
636
668
675
652
714
777
731
763
810
844
822
895
921
959
1009
1000
1024
1055
1083
1224
1313
1407
1655
1944
2468
3491
5015
8160
12963
20754
32620
48670
68705
94047
120435
145865
164299
172621
170323
156161
132140
104926
77449
52638
33897
20710
12098
6749
3938
2469
1713
1325
1096
1055
1027
1026
1223
1367
1700
2146
2671
3216
3595
3848
3888
3772
3320
2928
2426
2011
1497
1194
983
863
783
730
686
663
674
721
667
777
715
728
755
762
846
799
872
893
915
987
1109
1170
1271
1365
1612
1955
2347
3095
4169
5795
7591
10226
12398
14940
17162
19024
20866
23362
27807
34753
45702
60724
78426
97332
114141
127542
133368
132396
123935
112767
100444
88953
79926
73438
67990
62794
56911
49838
41142
32693
24794
18117
12466
8651
6154
4721
3916
3446
3316
3237
3205
3042
2817
2720
2323
2121
1780
1501
1204
1051
930
734
649
529
432
345
304
209
229
192
166
152
149
134
148
173
149
175
187
138
153
153
179
179
156
195
181
163
198
192
246
295
311
402
408
484
607
693
711
806
787
790
742
639
593
570
611
746
938
1464
2007
3097
4482
6166
8411
10627
12602
14213
15231
15510
14812
13092
11113
9031
6944
5115
3893
2915
2460
2346
2505
3001
3315
3751
4247
4547
4672
4596
4395
3986
3564
2967
2466
1953
1457
1111
801
547
428
315
270
183
183
205
221
231
291
361
432
566
556
632
681
598
608
564
468
356
262
213
154
97
60
44
50
26
29
18
16
17
19
18
12
12
18
11
9
13
6
8
10
8
8
10
14
12
9
14
10
13
11
8
10
11
5
8
13
6
12
9
13
9
10
6
5
12
10
6
8
6
6
12
9
6
7
8
5
5
9
8
8
4
11
9
8
4
5
10
9
6
5
5
9
4
6
13
14
10
6
15
10
13
16
20
18
40
37
32
32
28
31
22
26
24
30
21
9
8
9
5
3
8
7
9
8
2
7
4
4
2
4
4
6
3
6
6
5
3
7
3
5
4
3
3
8
6
9
5
6
2
7
6
9
3
3
8
6
9
10
6
5
3
8
3
7
4
6
2
6
7
5
2
9
3
8
5
5
6
7
10
6
5
6
7
3
3
5
7
4
10
6
5
6
7
10
11
14
15
14
21
28
21
21
31
32
20
27
21
17
16
13
12
10
9
11
8
6
5
8
7
3
12
2
7
5
6
7
5
12
8
5
6
9
4
8
9
7
13
6
1
9
8
6
9
13
11
13
12
24
20
28
59
47
78
122
123
141
147
197
203
216
232
227
212
217
213
160
130
88
98
68
75
58
46
32
26
18
28
36
35
31
30
31
32
43
50
45
55
38
38
41
47
50
42
45
38
39
49
53
62
51
43
43
50
56
84
83
92
76
133
126
123
150
155
169
181
164
177
172
193
165
188
172
157
175
152
155
136
127
147
152
138
170
160
193
198
213
264
295
329
351
382
384
397
398
389
382
354
321
277
256
240
238
154
151
148
126
97
81
74
51
43
32
20
27
23
19
22
16
22
26
18
18
14
15
12
11
14
10
9
10
9
4
10
7
12
4
6
7
6
4
4
4
8
4
5
5
6
12
7
14
5
14
17
23
30
25
33
30
36
45
64
65
101
95
92
92
95
118
160
147
166
155
172
194
187
197
166
181
225
193
207
192
171
184
153
135
117
115
101
86
87
90
71
55
52
46
40
26
32
26
25
22
29
26
14
23
20
18
12
9
9
6
9
15
5
3
2
4
2
3
3
1
2
1
1
1
3
4
5
4
2
2
4
2
5
3
1
2
0
4
0
3
4
3
4
5
7
6
4
8
7
7
8
7
10
12
21
22
21
22
29
29
28
34
36
50
48
39
40
35
39
35
27
24
27
33
27
16
27
24
20
21
23
26
16
11
17
13
11
10
9
16
8
7
9
4
5
2
5
2
3
1
4
4
3
1
8
2
3
3
0
2
1
2
4
0
1
0
0
1
2
0
1
0
1
0
0
0
0
0
0
1
0
2
1
0
0
0
1
0
0
0
1
0
0
0
0
0
1
1
1
0
2
2
1
2
3
1
2
1
1
1
1
3
0
2
1
1
1
0
3
0
5
0
2
1
0
1
0
2
1
0
0
0
0
0
0
1
2
1
1
1
2
0
0
0
1
0
0
0
0
1
0
1
2
0
1
0
1
0
0
0
0
0
0
0
0
1
1
1
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
1
0
0
0
0
0
0
1
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
1
1
1
0
0
0
0
0
1
0
0
0
0
0
0
0
1
0
0
0
0
2
0
0
0
0
1
1
0
1
0
0
1
0
2
0
1
1
0
0
1
0
0
0
0
0
1
0
1
1
0
2
0
0
1
0
2
0
0
1
0
2
1
0
0
1
2
1
0
1
0
1
1
2
0
3
1
1
1
0
2
0
0
0
0
0
0
2
0
0
1
0
0
0
0
1
0
0
0
0
0
0
0
0
1
1
0
0
0
0
0
0
2
0
1
2
2
0
0
0
0
0
1
0
0
1
0
2
0
2
0
3
1
0
0
0
1
0
1
1
3
1
0
0
1
0
0
1
1
0
2
0
0
0
1
2
1
1
0
1
1
1
2
0
0
1
0
0
2
1
0
0
1
0
1
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
1
0
0
0
1
0
0
0
0
0
0
0
2
1
0
0
0
0
0
1
0
0
1
0
0
0
1
0
0
0
0
0
1
0
0
0
1
1
0
0
1
0
0
0
0
0
1
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
File: C:\xrfdaten\2014KW30\chris\std20kev\std10078.spe
Start: 21.07.2014 18:32:43
Stop: 21.07.2014 18:34:23
BESSY: 298.5274938 298.5354909
IONI: 3.033556E-11 0.000000000030537
ScanMotor1 exp_x: 1 1
ScanMotor2 exp_y: 1 1
Live: 84111
Run: 100000
Dead: 16
ICR: 49270
OCR: 0
Accumulated BESSY: 59703.3359274
Accumulated Ioni: 6.09251103E-09
N Accumulation: 200
Normfactor:  3.9028411e+008
E_cal:      0.020000000
E_cal_offset:      0.000000
Comments: Pt
 
exp_x 91.2603988647461 91.2604
exp_y 65.3237991333008 65.3238
exp_z 53.8899993896484 53.89
exp_w 135 135
exp_Tisch 870.6 870.600001928
dcm_theta 0 0
det_y 52.85 52.85
dmm_energy 20 20
dcm_energy 23 -184976.74536672
S3_horSize 0.03 3.00000000000001E-02
S3_verSize 0.03 3.00000000000001E-02
$PEAKS:   
      1
  78   2      9.434   1715009.      1223.     227.83
