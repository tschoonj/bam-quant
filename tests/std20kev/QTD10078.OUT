     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ  AXIL IBM-PC V3.00                       10-08-2014   14:23:55  บ   
     บ  Spectrum: QTD10078.SPE                                     0.s บ   
     ศอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออผ   
     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ  Fitting Region: channels 100 -1980        3 iterations done    บ   
     บ  ChiSquare =   139.5   last change =   .02%    lambda= 1.E-05   บ   
     ฬอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออน   
     บ                        CALIBRATION  DATA                        บ   
     ฬออออออออออออออออหออออออออออออออออออออออออหอออออออออออออออออออออออน   
     บ                บ   Initial estimate     บ    Final estimate     บ   
     ฬออออออออออออออออฮออออออออออออออออออออออออฮอออออออออออออออออออออออน   
     บ  ZERO (eV)     บ      -29.5 ๑   100.0   บ     -29.5 ๑      .4   บ   
     บ  GAIN (eV/ch)  บ     20.217 ๑   2.000   บ    20.217 ๑    .001   บ   
     บ det NOISE (eV) บ       76.4 ๑    40.0   บ      76.4 ๑     2.6   บ   
     บ  FANO factor   บ       .136 ๑    .050   บ      .136 ๑    .001   บ   
     ศออออออออออออออออสออออออออออออออออออออออออสอออออออออออออออออออออออผ   
     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ                           PEAK  DATA                            บ   
     ฬอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออน   
     บ  # Line       E(KeV)  rel. int.   peak area   st. dev    chi-sq บ   
     บ               chan#   fwhm (eV)      backgr            tot. abs บ   
     ฬอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออน   
     บ                                                                 บ   
     บ  1 Pt-L1                            318574. ๑    861.           บ   
     บ    L1M3       11.231    .39992      127404. ๑    345.   1608.7  บ   
     บ              556.984    196.19         255.           9.83E-01  บ   
     บ    L1M2       10.854    .33846      107824. ๑    292.    549.9  บ   
     บ              538.336    193.37         268.           9.82E-01  บ   
     บ    L1N3       13.361    .11195       35663. ๑     97.    867.3  บ   
     บ              662.341    211.39         185.           9.87E-01  บ   
     บ    L1N2       13.270    .08745       27859. ๑     77.   1082.6  บ   
     บ              657.840    210.76         188.           9.86E-01  บ   
     บ    L1O3       13.822    .03448       10984. ๑     33.     94.1  บ   
     บ              685.144    214.53         191.           9.86E-01  บ   
     บ    L1M5       11.758    .01663        5298. ๑     21.     12.0  บ   
     บ              583.051    200.06         236.           9.85E-01  บ   
     บ    L1M4       11.676    .01112        3542. ๑     18.     18.5  บ   
     บ              578.995    199.46         239.           9.85E-01  บ   
     บ                                                                 บ   
     บ  2 Pt-L2                           1459028. ๑   1371.           บ   
     บ    L2M4       11.071    .79220     1155847. ๑   1086.    910.8  บ   
     บ              549.070    195.00         260.           9.83E-01  บ   
     บ    L2N4       12.943    .16358      238668. ๑    225.   2079.0  บ   
     บ              641.665    208.49         198.           9.86E-01  บ   
     บ    L2M1        9.975    .02146       31310. ๑     34.    247.5  บ   
     บ              494.858    186.65         269.           9.77E-01  บ   
     บ    L2N1       12.552    .00561        8179. ๑     16.     21.7  บ   
     บ              622.325    205.75         210.           9.84E-01  บ   
     บ    L2O4       13.271    .01525       22250. ๑     25.   1082.6  บ   
     บ              657.889    210.77         188.           9.86E-01  บ   
     บ    L2O1       13.173    .00110        1610. ๑     14.   1405.2  บ   
     บ              653.042    210.09         191.           9.86E-01  บ   
     บ    L2M3       10.627    .00080        1164. ๑     17.    858.1  บ   
     บ              527.108    191.66         276.           9.81E-01  บ   
     บ                                                                 บ   
     บ  3 Pt-L3                           2158140. ๑   1539.           บ   
     บ    L3M5        9.442    .71362     1540102. ๑   1098.    227.8  บ   
     บ              468.494    182.45         288.           9.73E-01  บ   
     บ    L3M4        9.362    .08105      174906. ๑    126.    159.5  บ   
     บ              464.537    181.81         291.           9.72E-01  บ   
     บ    L3N5       11.251    .14054      303309. ๑    217.   1570.4  บ   
     บ              557.973    196.34         254.           9.84E-01  บ   
     บ    L3M1        8.268    .03904       84257. ๑     63.    160.6  บ   
     บ              410.423    172.84         334.           9.60E-01  บ   
     บ    L3N1       10.842    .00978       21097. ๑     22.    545.9  บ   
     บ              537.742    193.28         269.           9.82E-01  บ   
     บ    L3O1       11.462    .00192        4144. ๑     16.    835.0  บ   
     บ              568.410    197.89         246.           9.84E-01  บ   
     บ    L3O5       11.561    .01405       30324. ๑     27.    205.1  บ   
     บ              573.307    198.62         242.           9.85E-01  บ   
     บ                                                                 บ   
     บ  4 Pile_up                           17510. ๑    135.           บ   
     บ    L3O5       20.505    .25605        4529. ๑     36.      7.8  บ   
     บ             1015.700    255.85          55.           9.41E-01  บ   
     บ    L3N1       18.868    .18984        3358. ๑     27.      4.6  บ   
     บ              934.721    246.37          81.           9.69E-01  บ   
     บ    L3O1       20.679    .09533        1686. ๑     15.      1.7  บ   
     บ             1024.311    256.84          52.           9.37E-01  บ   
     บ    L3O5       22.142    .08634        1527. ๑     13.      6.4  บ   
     บ             1096.680    264.99          37.           8.99E-01  บ   
     บ               22.316    .06429        1137. ๑     11.      2.8  บ   
     บ             1105.291    265.94          35.           8.94E-01  บ   
     บ    L2N1       22.377    .05287         935. ๑      9.      2.2  บ   
     บ             1108.296    266.27          35.           8.92E-01  บ   
     บ    L3M5       24.014    .03566         631. ๑      7.      3.3  บ   
     บ             1189.276    275.07          19.           8.39E-01  บ   
     บ    L3N5       20.286    .02852         504. ๑      9.     69.2  บ   
     บ             1004.870    254.60          58.           9.45E-01  บ   
     บ    L3M1       22.739    .01934         342. ๑      6.      3.7  บ   
     บ             1126.231    268.25          31.           8.81E-01  บ   
     บ    L3N1       21.923    .01923         340. ๑      7.      7.4  บ   
     บ             1085.849    263.78          40.           9.05E-01  บ   
     บ               17.702    .01865         330. ๑     11.      1.1  บ   
     บ              877.054    239.39         104.           9.81E-01  บ   
     บ               24.188    .01327         235. ๑      5.      3.9  บ   
     บ             1197.887    275.99          18.           8.33E-01  บ   
     บ               24.377    .01304         231. ๑      4.      4.1  บ   
     บ             1207.211    276.98          17.           8.26E-01  บ   
     บ               19.339    .01258         222. ๑      9.      7.2  บ   
     บ              958.034    249.13          72.           9.63E-01  บ   
     บ               22.490    .01197         212. ๑      6.      2.2  บ   
     บ             1113.902    266.89          34.           8.88E-01  บ   
     บ               20.983    .00763         135. ๑      7.      1.3  บ   
     บ             1039.349    258.55          48.           9.30E-01  บ   
     บ               22.097    .00716         127. ๑      6.      6.6  บ   
     บ             1094.460    264.74          38.           9.00E-01  บ   
     บ               19.409    .00694         123. ๑      9.     10.8  บ   
     บ              961.488    249.54          71.           9.61E-01  บ   
     บ               22.620    .00515          91. ๑      6.      3.3  บ   
     บ             1120.328    267.60          32.           8.84E-01  บ   
     บ               24.551    .00485          86. ๑      4.      3.3  บ   
     บ             1215.822    277.89          16.           8.20E-01  บ   
     บ               19.513    .00468          83. ๑      8.     18.5  บ   
     บ              966.645    250.15          70.           9.60E-01  บ   
     บ               21.046    .00468          83. ๑      7.       .9  บ   
     บ             1042.468    258.91          47.           9.28E-01  บ   
     บ               23.795    .00397          70. ๑      5.      1.4  บ   
     บ             1178.445    273.91          21.           8.46E-01  บ   
     บ               25.886    .00368          65. ๑      3.      4.9  บ   
     บ             1281.871    284.79           8.           7.73E-01  บ   
     บ               26.249    .00269          48. ๑      3.      4.7  บ   
     บ             1299.806    286.64           7.           7.60E-01  บ   
     บ               21.211    .00260          46. ๑      7.       .9  บ   
     บ             1050.629    259.83          45.           9.24E-01  บ   
     บ               23.256    .00243          43. ๑      5.      1.7  บ   
     บ             1151.774    271.03          26.           8.64E-01  บ   
     บ               21.159    .00196          35. ๑      7.       .9  บ   
     บ             1048.057    259.54          46.           9.25E-01  บ   
     บ               22.794    .00192          34. ๑      6.      3.6  บ   
     บ             1128.939    268.54          30.           8.79E-01  บ   
     บ               21.986    .00181          32. ๑      6.      7.5  บ   
     บ             1088.956    264.13          39.           9.03E-01  บ   
     บ               21.220    .00174          31. ๑      7.      1.0  บ   
     บ             1051.079    259.88          45.           9.24E-01  บ   
     บ               24.893    .00164          29. ๑      4.      1.9  บ   
     บ             1232.754    279.68          13.           8.08E-01  บ   
     บ               24.158    .00145          26. ๑      4.      3.8  บ   
     บ             1196.380    275.83          18.           8.34E-01  บ   
     บ               19.120    .00140          25. ๑      9.      5.3  บ   
     บ              947.203    247.85          76.           9.66E-01  บ   
     บ               22.796    .00132          23. ๑      5.      3.6  บ   
     บ             1129.036    268.55          30.           8.79E-01  บ   
     บ               23.623    .00122          22. ๑      5.      1.0  บ   
     บ             1169.935    272.99          23.           8.52E-01  บ   
     บ               21.704    .00107          19. ๑      7.      5.1  บ   
     บ             1075.018    262.58          43.           9.11E-01  บ   
     บ               24.492    .00106          19. ๑      4.      3.7  บ   
     บ             1212.924    277.58          16.           8.22E-01  บ   
     บ                                                                 บ   
     ศอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออผ   
                                                                           
     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ                         FILTER BACKGROUND                       บ   
     ศอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออผ   
                                                                           
